api/routes/product.js

```diff
@@ -2,7 +2,6 @@ const router = require('express').Router();
 const product = require('../controllers/product');
 
 router.get('/products', product.paginatedResults);
-router.get('/filteredProducts', product.filterProducts);
 
 router.post('/addProduct', product.addProduct);
```

Comment:
Just to keep the REST convention you can remove the add, edit, delete from the url and keep only `/product`. The url should be the same only the method should be different.
```suggestion
router.post('/product', product.addProduct);
```